python/pyspark/sql/functions.py

```diff
@@ -828,7 +828,7 @@ def lead(col, count=1, default=None):
     :param default: default value
```

Comment:
Maybe also document the change in param name in the docstring here?

Reply:
@holdenk Done!